app/models/public_body.rb

```diff
@@ -202,6 +202,18 @@ def set_api_key!
     self.api_key = SecureRandom.base64(33)
   end
 
+  def self.find_by_name(name)
+    PublicBody.joins(:translations).
+      where("public_body_translations.name = ?", name).
+       first
+  end
+
+  def self.find_by_url_name(url_name)
```

Comment:
We need to do this because these no longer exist now that we've dropped the columns on this table.

- Commit message needs to indicate why these are required.
- It surprises me that globalize doesn't reimplement these. Maybe they've moved to a gem or something?
- Doesn't look like these take locale in to account
- These need testing – specifically around locale fallbacks etc
- Probably don't need the class prefix. just start with `joins`
- Can probably use hash keys for a bit nicer syntax while we're at it `where(:public_body_translations => { :url_name => url_name } )`